client/src/components/HomeScreen.tsx

```diff
@@ -291,12 +306,43 @@ const CreateAudit: React.FC = () => {
               <p>Audit type</p>
               <RadioGroup
                 name="auditType"
-                onChange={e =>
+                onChange={e => {
                   setFieldValue('auditType', e.currentTarget.value)
-                }
+                  switch (values.auditType) {
+                    case 'BALLOT_POLLING': {
+                      setFieldValue('auditMathType', 'BRAVO')
+                      break
+                    }
+                    case 'BALLOT_COMPARISON': {
+                      setFieldValue('auditMathType', 'SUPERSIMPLE')
+                      break
+                    }
+                    default: {
+                      setFieldValue('auditMathType', 'MACRO')
+                      break
+                    }
+                  }
```

Comment:
Two things:
1. I'm a little sketched out by calling `setFieldValue('auditType', e.currentTarget.value)` and then immediately trying to access `values.auditType` in the next line. Maybe it works, but it seems suspicious/not clear to me, since I don't know how `setFieldValue` works. Also, I would expect that `setFieldValue` triggers a re-render each time it's called. I think it might be better to use Formik's `setValues` and set both auditType and auditMathType in the same call, so that only one re-render is triggered.
2. I generally like to make all cases explicit instead of using default (in case another case gets added later). one way to do that is with an object.

So overall I might try this:

```ts
const auditType = e.currentTarget.value
const auditMathType = {
  BALLOT_POLLING: "BRAVO",
  BALLOT_COMPARISON: "SUPERSIMPLE",
  BATCH_COMPARISON: "MACRO",
}[auditType]
setValues({ ...values, auditType, auditMathType })
```